tests/publiccloud/run_ltp.pm

```diff
@@ -48,38 +48,13 @@ sub instance_log_args
 sub upload_ltp_logs
 {
     my ($self) = @_;
-    my $ltp_testsuite = get_required_var('LTP_COMMAND_FILE');
-    my $log_file = Mojo::File::path('ulogs/result.json');
     record_info('LTP Logs', 'upload');
-    upload_logs("$root_dir/result.json", log_name => $log_file->basename, failok => 1);
```

Comment:
Also, you removed this whole log `result.json`  upload that was added in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15774, after discussed with kernel team, that considered it useful to debug some issues.